### ISSUE TITLE
[Feature] Option to disable morph 

### ISSUE THREAD

**Issue by mataslib** [isaacHagoel/svelte-dnd-action#270]
Add option to disable dragged item morph to zone dimensions. Effect is nice, but there may be cases, when you don't want morph to happen. At least our UX designer designed D&D like so. When morph is ussed, some items may be hidden under full width morphed item, also morph animation in combination with another "placeholder" shadow item animation may be disturbing. It may also be better option in combination with center on cursor => dragging smaller box.

In short: I have some palette of components (zone) to be dragged to another zone and I want these components to preserve their original dimensions.

one image === thousand words:

![image](https://user-images.githubusercontent.com/5154014/113488306-87065100-94bd-11eb-801d-21e4c89c87c9.png)

**Comment by isaacHagoel** [comment]
I merged the PR but just FYI - you don't show up in the contributors list. it looks like your commits are not properly associated with your github account (I think it has to do with your local git settings). You might want to fix that.

**Comment by mataslib** [comment]
Hello isaac, thanks for info, should be fine from now on. 👍 I did not have local git email in github emails list.
